Example/src/AnimatedStyleUpdateExample.tsx

```diff
@@ -36,7 +36,13 @@ function AnimatedStyleUpdateExample(): React.ReactElement {
       <Button
         title="toggle"
         onPress={() => {
-          randomWidth.value = Math.random() * 350;
+          const newValue = Math.random() * 350;
+          randomWidth.value = newValue;
+          console.log('HERE', randomWidth.value, newValue);
+          // const hh = doRandomStuff(randomWidth);
+          // console.log('AFTER', randomWidth.value, newValue, hh);
+          setTimeout(() => console.log('HHH', randomWidth.value), 50);
+          // setTimeout(() => doRandomStuff(randomWidth), 150);
```

Comment:
Can we move it to a separate example?

Reply:
sure, it wasn't meant to be here actually, was just making some temporary changes for testing purposes and accidently commited these

Reply:
fixed here: https://github.com/software-mansion/react-native-reanimated/pull/5513/commits/04b30b8fa4450770299e5813098ed842f4d9af3b